t/24-worker-engine.t

```diff
@@ -58,7 +58,7 @@ subtest 'isotovideo version' => sub {
 
     my $isotovideo = path($FindBin::Bin)->child('fake-isotovideo.pl');
     my $worker2 = OpenQA::Worker->new({apikey => 'foo', apisecret => 'bar', instance => 1, isotovideo => $isotovideo});
-    is($worker2->isotovideo_interface_version, 15, 'isotovideo version set from os-autoinst');
+    is($worker2->isotovideo_interface_version, WEBSOCKET_API_VERSION, 'isotovideo version set from os-autoinst');
```

Comment:
We should consequently remove `isotovideo_interface_version` and the entire test as well.

Reply:
Agree. If we don't need os-autoinst version, why we return `WEBSOCKET_API_VERSION`? is it used somewhere?